packages/react/components/modal/Modal.tsx

```
@@ -52,9 +53,11 @@ const Modal = React.forwardRef<ModalRef, ModalProps>(
 
     const handleClose = React.useCallback(
       (onCloseFunc: ClickEvent | undefined, e: OnClickEvent) => {
-        setDisplay(false)
-        refBtnModal.current && refBtnModal.current.focus()
-        if (onCloseFunc) onCloseFunc(e)
+        if (!unClosable) {
+          setDisplay(false)
+          refBtnModal.current && refBtnModal.current.focus()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)